configure.ac

```diff
@@ -462,6 +462,25 @@ fi
 AC_SUBST(COCKPIT_WSINSTANCE_USER)
 AC_SUBST(COCKPIT_WSINSTANCE_GROUP)
 
+# admin users group
+AC_ARG_WITH([admin-group],
+            [AC_HELP_STRING([--with-admin-group=GROUP],
+                            [system group to which admin users belong])],
+            [admin_group=$withval],
+            [
+              AC_MSG_CHECKING([for system group to which admin users belong])
+              CANDIDATE_GROUPS="wheel sudo root"
```

Comment:
I basically did that so that we'd have a reasonable fallback instead of failing the build.  The impact of this is that the file would end up owned by `root:root` which is what it was already owned by....

Reply:
That's a good point, right. I failed to appreciated at that point that it's just a file permission.